README.md

```diff
@@ -1 +1,122 @@
-# fedoramy.github.io
\ No newline at end of file
+# fedoramy.github.io
+
+## Set up dev env
+I use virtualenv is a tool to create isolated Python environments.
+```
+$ pip install virtualenv
+```
+
+fork this repo and navigate into it and run
+```
+$ python -m venv env
+```
+it will create a virtual environment for you
+
+To begin using the virtual environment, it needs to be activated:
+```
+$ source env/bin/activate
+```
+
+Now install the package dependencies.
+```
+$ pip install -r requirements.txt
+```
+
```

Comment:
We can move this to the bottom, or doesn't need it all. What people will interested is the build instruction.